output/github.go

```diff
@@ -0,0 +1,94 @@
+package output
+
+import (
+	"fmt"
+	"io"
+)
+
+// GitHub represents an Outputter that outputs
+// results in GitHub workflow format.
+type GitHub struct {
+	Writer io.Writer
```

Comment:
🤔 I don't think we need to expose this, let's do
```suggestion
	writer io.Writer
```

...but it looks like this is consistent with the other plugins, so fine -- it's not a big deal anyways.

Reply:
I started off just as the other output formats, but you're right, io.Writer can be private.